src/components/user/myProfile/FriendList.js

```diff
@@ -15,14 +16,14 @@ export default function MyProfile() {
 	};
 
 	return (
-        <div>
-		<h1>Friends list:  </h1>
+        <div id="friendListContainer">
+		<h1>Friends list:</h1>
 		
 		<div className="FriendList">
 		<List src="user.foaf_knows" >
 		{
 			(item, i) =>
-				<Card key={uniqueKey.toString()} onClick={() => openProfile(`${item}` )}>
+				<Card id={uniqueKey.toString()} onClick={() => openProfile(`${item}` )}>
```

Comment:
Here it needs to be key, not id. Key propierty is the one that must be unique for each element of a map.